src/test/java/org/opensearch/neuralsearch/query/HybridQueryBuilderTests.java

```diff
@@ -426,7 +426,7 @@ public void testFromXContent_whenMultipleSubQueries_thenBuildSuccessfully() {
         assertEquals(2, queryTwoSubQueries.queries().size());
         assertTrue(queryTwoSubQueries.queries().get(0) instanceof NeuralQueryBuilder);
         assertTrue(queryTwoSubQueries.queries().get(1) instanceof TermQueryBuilder);
-        assertEquals(10, queryTwoSubQueries.paginationDepth());
+        assertEquals(10, queryTwoSubQueries.paginationDepth().intValue());
```

Comment:
do we have a test scenario when pagination_depth is not set/is null?

Reply:
Well this test belongs to the exact same case. So when the value is not sent in the request then it will pick the default value and this is the exact thing what this assert condition validates. If you see this test case I haven't set the value of pagination_depth explicitly. So it is going null.